Mimir.MongoDB/Json/Extensions/MimirBsonDocumentExtensions.cs

```diff
@@ -1,26 +1,24 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
 using Mimir.MongoDB.Bson;
 using Mimir.MongoDB.Json.Converters;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Converters;
 
 namespace Mimir.MongoDB.Json.Extensions;
 
 public static class MimirBsonDocumentExtensions
 {
-    public static readonly JsonSerializerSettings JsonSerializerSettings = new()
+    public static readonly JsonSerializerOptions JsonSerializerOptions = new()
     {
-        Formatting = Formatting.None,
-        NullValueHandling = NullValueHandling.Ignore,
-        ReferenceLoopHandling = ReferenceLoopHandling.Ignore,
+        WriteIndented = false,
+        DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull,
         Converters =
-        [
+        {
             new BigIntegerJsonConverter(),
             new MaterialAndIntDictionaryJsonConverter(),
-            new StringEnumConverter(),
-        ],
-        ContractResolver = new IgnoreBencodedContractResolver(),
```

Comment:
To replace `IgnoreBencodedContractResolver`, you have to attach `JsonIgnore` for fields starting with `Bencoded`. You can see the logic at https://github.com/planetarium/mimir/blob/2e0f4dbdc443e86725ee2b674cb0d73ddfc459cf/Mimir.MongoDB/Json/Converters/IgnoreBencodedContractResolver.cs